data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/SortingConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public class SortingConfiguration {
+
+    @JsonProperty("sort_key")
+    private List<String> sortKey;
+
+    @JsonProperty("order")
+    private String order;
```

Comment:
We should validate that that if this is set, the value is either `asc` or `desc`. See here for an example of validating (https://github.com/opensearch-project/data-prepper/blob/42cf9d09d71351b38baacb2dd9c8aad387625ea2/data-prepper-plugins/http-source/src/main/java/org/opensearch/dataprepper/plugins/source/loghttp/HTTPSourceConfig.java#L132)

You could also make this an enum that would handle the validation while trying to convert the String to an enum, but I am fine with the simple validation approach